### ISSUE TITLE
MiKTeX 2.9 - User/administrator updates are out-of-sync

### ISSUE THREAD

**Issue by EDVBeratung-Zimmermann** [MiKTeX/miktex#518]
When starting the Console I receive this message and the application stops. Also I start the Console in Admin-Mode I receive this message.
Additionally I am not able to remove MiKTeX from Windows.

Regards
Thomas Zimmermann

**Comment by edocevoli** [comment]
Please follow the [contribution guidelines](https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md) when submitting an issue ticket.

**Comment by EDVBeratung-Zimmermann** [comment]
Gentlemen,

 

thank your for your quick reply. However

 

1.	I have installed the latest MiKTeX updates
2.	The bugreport was selfcontained with no links
3.	There was a step-by-step reproduction instruction: I started the MiKTeX console and received an error message: ”User/administrator udates are out-of-sync”
4.	There was no need for a configuration report.
5.	There were no input files to attach
6.	I found no log files

 

Before I wrote the bug report

1.	I checked the MiKTeX FAQ, especially the lin within the erreor message. This did not fix the problem
2.	I wrote down the one liner „User/administrator updates are out-of-syn”
3.	I performed an internet search with no results

 

Okay – I forgot some more information about the Installation.

I am working with Windows 10 pro, current version and MiKTeX 2.9.

 

To sum it up: the MiKTeX console does not work – neither in user nor in admin mode and I am not able do de-install/remove the whole installation.

For the moment I am rather disappointed.

 

Kind regards

Thomas Zimmermann

 

Von: Christian Schenk <notifications@github.com> 
Gesendet: Monday, 27 April 2020 21:02
An: MiKTeX/miktex <miktex@noreply.github.com>
Cc: EDVBeratung-Zimmermann <kontakt@edv-beratung.familiezimmermann.de>; Author <author@noreply.github.com>
Betreff: Re: [MiKTeX/miktex] MiKTeX 2.9 - User/administrator updates are out-of-sync (#518)

 

Please follow the contribution guidelines <https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md>  when submitting an issue ticket.

**Comment by EDVBeratung-Zimmermann** [comment]
Gentlemen,

thank your for your quick reply. However

1.	I have installed the latest MiKTeX updates
2.	The bugreport was selfcontained with no links
3.	There was a step-by-step reproduction instruction: I started the MiKTeX console and received an error message: ”User/administrator udates are out-of-sync”
4.	There was no need for a configuration report.
5.	There were no input files to attach
6.	I found no log files

Before I wrote the bug report
1.	I checked the MiKTeX FAQ, especially the lin within the erreor message. This did not fix the problem
2.	I wrote down the one liner „User/administrator updates are out-of-syn”
3.	I performed an internet search with no results

Okay – I forgot some more information about the Installation.
I am working with Windows 10 pro, current version and MiKTeX 2.9.

To sum it up: the MiKTeX console does not work – neither in user nor in admin mode and I am not able do de-install/remove the whole installation.
For the moment I am rather disappointed.

Kind regards
Thomas Zimmermann

**Comment by edocevoli** [comment]
Thank you. But without log files I will not be able to investigate the issue.

**Comment by edocevoli** [comment]
It would be helpful, if you make a screenshot of the error message.

**Comment by EDVBeratung-Zimmermann** [comment]
Hello – where does miktex-console.exe saves  the logfile ?

I have no clue.

 

 

Von: Christian Schenk <notifications@github.com> 
Gesendet: Tuesday, 28 April 2020 21:25
An: MiKTeX/miktex <miktex@noreply.github.com>
Cc: EDVBeratung-Zimmermann <kontakt@edv-beratung.familiezimmermann.de>; Author <author@noreply.github.com>
Betreff: Re: [MiKTeX/miktex] MiKTeX 2.9 - User/administrator updates are out-of-sync (#518)

 

Thank you. But without log files I will not be able to investigate the issue.

**Comment by EDVBeratung-Zimmermann** [comment]
Please find attached the screenshot.

 

 

Von: Christian Schenk <notifications@github.com> 
Gesendet: Tuesday, 28 April 2020 21:29
An: MiKTeX/miktex <miktex@noreply.github.com>
Cc: EDVBeratung-Zimmermann <kontakt@edv-beratung.familiezimmermann.de>; Author <author@noreply.github.com>
Betreff: Re: [MiKTeX/miktex] MiKTeX 2.9 - User/administrator updates are out-of-sync (#518)

 

It would be helpful, if you make a screenshot of the error message.

**Comment by edocevoli** [comment]
There is no screenshot 😞   Please:
- don't use the E-Mail interface to send files; it simply does not work; besides that: it exposes your E-Mail address to the world
- instead: visit the issue page on GitHub

The location of your log files can be seen in MiKTeX Console. The common locations are:

```
%LOCALAPPDATA%\MiKTeX\2.9\miktex\log
%PROGRAMDATA%\MiKTeX\2.9\miktex\log
```

It is recommended that you create a Zip file which you attach to the issue.

**Comment by EDVBeratung-Zimmermann** [comment]
Thank you for the hint. Please find attached the Error-Message und the log-File from C:\Users\thoma\AppData\Local\MiKTeX\2.9\miktex\log
[miktex-console.log](https://github.com/MiKTeX/miktex/files/4552770/miktex-console.log)

![2020-04-29](https://user-images.githubusercontent.com/38407000/80617053-e6ecb000-8a41-11ea-8ef0-6c464f357a4f.png)

**Comment by edocevoli** [comment]
Thank you. But
- we need all log files (from both locations)
- have you tried the mentioned workaround?

**Comment by EDVBeratung-Zimmermann** [comment]
Okay – regarding the workaround – it did not work

 

Von: Christian Schenk <notifications@github.com> 
Gesendet: Wednesday, 29 April 2020 20:04
An: MiKTeX/miktex <miktex@noreply.github.com>
Cc: EDVBeratung-Zimmermann <kontakt@edv-beratung.familiezimmermann.de>; Author <author@noreply.github.com>
Betreff: Re: [MiKTeX/miktex] MiKTeX 2.9 - User/administrator updates are out-of-sync (#518)

 

Thank you. But

*	we need all log files (from both locations)
*	have you tried the mentioned workaround?

**Comment by EDVBeratung-Zimmermann** [comment]
C:\ProgramData\MiKTeX\2.9\miktex\log
[log.zip](https://github.com/MiKTeX/miktex/files/4553525/log.zip)
[miktex-console_admin.log](https://github.com/MiKTeX/miktex/files/4553527/miktex-console_admin.log)
[miktex-fc-cache_admin.log](https://github.com/MiKTeX/miktex/files/4553528/miktex-fc-cache_admin.log)
[mkfntmap_admin.log](https://github.com/MiKTeX/miktex/files/4553529/mkfntmap_admin.log)
[mpmcli_admin.log](https://github.com/MiKTeX/miktex/files/4553530/mpmcli_admin.log)
[initexmf_admin.log](https://github.com/MiKTeX/miktex/files/4553531/initexmf_admin.log)

**Comment by EDVBeratung-Zimmermann** [comment]
C:\ProgramData\MiKTeX\2.9\miktex\log
[log.zip](https://github.com/MiKTeX/miktex/files/4553537/log.zip)

**Comment by edocevoli** [comment]
The workaround is "stay in MiKTeX user mode and check for updates". Have you done that According to `miktex-console.log`, the last user update check was done on on April 13:

```
2020-04-13 17:18:20,940+0200 INFO  mainwindow - found 0 updates
```

The last admin update was also on April 13, but at a later time:

```
2020-04-13 17:18:35,334+0200 INFO  mainwindow - found 6 updates
2020-04-13 17:19:09,801+0200 INFO  mainwindow - installing updates...
```

So yes, admin/user updates are out-of-sync. Just do what the message box recommends: check for updates as user.

I am out now.
